Library/Formula/poco.rb

```diff
@@ -38,6 +39,11 @@ def install
     mkdir "macbuild" do
       system "cmake", buildpath, *args
       system "make", "install"
+      if build.with? "static"
+      	args << "-DPOCO_STATIC=ON" 
```

Comment:
Can this not be inlined to the prior block rather than enforcing doing the entire build again?